fullstop-web/fullstop-violations-rest/src/main/java/org/zalando/stups/fullstop/web/model/Violation.java

```diff
@@ -109,7 +109,7 @@ public void setComment(String comment) {
         this.comment = comment;
     }
 
-    @ApiModelProperty(value = "")
+    @ApiModelProperty(value = "", dataType = "DateTime", example = "2015-05-21T10:24:47.788-02:00")
```

Comment:
example and datatype are not really working. I should find another solution for that